src/rt-app.c

```diff
@@ -1093,7 +1093,7 @@ static void set_thread_param(thread_data_t *data, sched_data_t *sched_data)
 		return;
 
 	/* if no policy is specified, reuse the previous one */
-	if ((sched_data->policy == same) && data->curr_sched_data)
+	if ((sched_data->policy == none) && data->curr_sched_data)
```

Comment:
replacing the -1 value by the enum one makes sense but I really prefer keeping same instead of none because that really what it means same as previous phase

Reply:
OK, will change it.